.github/workflows/release.yml

```diff
@@ -279,11 +279,23 @@ jobs:
           # Add additional line before the content
           RELEASE_NOTES="✅ New *${{ github.event.repository.name }}* Release [${{ needs.version-tag.outputs.version }}](${RELEASE_URL}) $(date)\n\n${RELEASE_NOTES}\n\nCommit Details:\n${new_fix_notes}"
 
-          # Combine existing content with new fix notes
-          updated_content="${existing_content}<br/><br/>${RELEASE_NOTES}"
```

Comment:
- When replacing markdown links with Confluence links, consider the impact on the readability and usability of the content. Make sure the new format is consistent with the rest of the system.